network/templates/includes/station_edit.html

```diff
@@ -53,11 +53,21 @@ <h4 class="modal-title" id="StationModal">Edit Ground Station</h4>
               <input class="form-control"
                      id="location"
                      type="text"
-                     placeholder="Geocoded location here"
+                     placeholder="Geocoded address here"
                      disabled>
             </div>
           </div>
           <div class="form-group">
+            <label for="qthlocator" class="col-sm-2 control-label">QTH Locator</label>
+            <div class="col-sm-10">
+              <input class="form-control"
+                     id="qthlocator"
+                     type="text"
+                     name="qthlocator"
+                     value="{{ form.qthlocator.value|default_if_none:"Geocoded gridsquare here" }}">
```

Comment:
You could add a [`readonly`](http://www.w3.org/TR/html5/forms.html#the-readonly-attribute) value here to make it immutable. See